### PR TITLE
Fix GitHub warnings related to `::set-output` deprecation

### PR DESCRIPTION
**Pull request by kachkaev**
## 🌟 What is the purpose of this PR?

This PR removes noise from CI output:

Screenshot 2022-10-14 at 15 57 50

## 🔗 Related links

- [Context (blog post)](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

## Known issues

There are still warnings, all of them are related to upstream code though:

Screenshot 2022-10-24 at 13 19 23

Upstream issue: https://github.com/Asana/create-app-attachment-github-action/issues/16

Other warnings should be fixed by updating GitHub actions, I'll do this via Renovate bot. This will keep PRs focused.

## 📜 Does this require a change to the docs?

No

## ❓ How to test this?

Check CI

## 📹 Demo

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203225227202708

### CODE REVIEW

File: .github/scripts/rust/setup.py [hunk 1]
```diff
@@ -217,17 +217,22 @@ def main():
     changed_parent_crates = filter_parent_crates(changed_crates)
     changed_docker_crates = filter_for_docker_crates(changed_parent_crates)
 
-    output_matrix("lint", changed_parent_crates)
-    output_matrix("test", changed_parent_crates, profile=["development", "production"])
-    output_matrix("docker", changed_docker_crates, profile=["production"])
+    github_output_file = open("/tmp/github_output.txt", "w")
```
[the remark this answers is not in the document]
Comment (kachkaev) [reply .github/scripts/rust/setup.py]: I used env var for simplicity

[the remark this answers is not in the document]
Comment (kachkaev) [reply .github/scripts/rust/setup.py]: CI passes 🎉

File: packages/engine/lib/execution/src/runner/python/setup.sh [hunk 2]
```diff
@@ -73,5 +73,6 @@ fi
 echo "Running setup.py"
 # Also compile Cython. Can be done either after
 # or before compiling Rust.
+export GITHUB_OUTPUT_FILE_PATH=github_output.txt
```
Comment (TimDiekmann) [inline packages/engine/lib/execution/src/runner/python/setup.sh:76]: This is not related to GitHub but is only used for the engine. I doubt this is needed

Comment (kachkaev) [reply packages/engine/lib/execution/src/runner/python/setup.sh]: I see, you are right. I removed this line.

**Review by TimDiekmann — APPROVED** [review]

**Pull request merged**